daily_spellbook/models/op/governance/optimism/op_governance_optimism_delegates.sql

```diff
@@ -1,6 +1,7 @@
 {{ config(
      schema = 'op_governance_optimism'
         , alias = 'delegates'
+        , tags=['prod_exclude'] -- Due to duplicates
```

Comment:
@kaiblade this model has duplicates which is causing tests to fail. Could you please have a look? This will be excluded in the meantime. I did some debugging here https://dune.com/queries/3796867

Reply:
i believe @kaiblade was working with @chuxinh and @MSilb7 here too, fyi to all

Reply:
Sorry about this. I will dedupe the model as soon as possible.

Reply:
I have deduped the model, here is the associated PR #6071.